lib/parallel_tests/cli.rb

```diff
@@ -167,9 +167,17 @@ def parse_options!(argv)
         options[:non_parallel] = true
       end
 
-      abort "Pass files or folders to run" if argv.empty? && !options[:execute]
+      extra_test_options, files = parse_additional_test_options(argv)
 
-      options[:files] = argv
+      unless options[:execute]
+        abort "Pass files or folders to run" unless files
+        options[:files] = files
+      end
+
+      if extra_test_options
+        test_option_array = Array(options[:test_options]) + extra_test_options
+        options[:test_options] = test_option_array.join(' ')
```

Comment:
a small bug when using with options that include spaces:
``--foo 'bar baz'` -> ['--foo', 'bar baz'] -> '--foo bar baz'`

```
require 'shellwords'
['foo', 'bar baz'].shelljoin
-> "foo bar\\ baz"
```